_rules/SC1-2-Video-description-track.md

```diff
@@ -17,18 +12,19 @@ test_aspects:
 authors:
 - Wilco Fiers
 - Brian Bors
+---
 
 ## Test Procedure
 
 ### Applicability
 
-The rule applies to any [non-streaming][] video element [visible in the page][] where the video contains [audio][] and a `track` element with a `kind="description"` attribute.
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video contains audio and a `track` element with a `kind="description"` attribute.
```

Comment:
the keyword for the kind attribute is descriptions, not description

Reply:
Thanks! Nice catch.